src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+}
+
+.big-bar {
+  height: 68px;
+  border: 1px solid rgb(230, 228, 228);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+.label {
+  display: flex;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 25px;
+  right: 18px;
+  top: 24px;
+}
+
+.big-bar .input-big {
+  border: 0;
+  outline: 0;
+  width: 100%;
+  font-size: 16px;
+  padding: 24px 62px;
+}
+
+.big-bar:hover {
+  box-shadow: 1px 5px 5px gray;
+}
+
+.big-bar input::placeholder {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
```

Comment:
```suggestion
  font-family: inherit;
```
fix everywhere, you already have font-family in html selector

Reply:
When I try to inherit the font-family - nothing works correctly. After inherit added it looks not like in the task example.